### DISCUSSION TITLE
D3Trees error message

### DISCUSSION THREAD

**Discussion by Etwari3** [JuliaPOMDP/POMDPs.jl#488] [Debugging Help]
Hello,
 
I got an error message when I tested D3Trees. Can you please help?

![image](https://github.com/JuliaPOMDP/POMDPs.jl/assets/78047777/df3d2f25-e295-4737-b083-c04754065654)

Thank you.

**Comment by zsunberg** [comment]
What's the output of `keys(info)`?

**Reply by Etwari3** [reply]
hello zsunberg,
It may be referring to upper and lower bounds, mu, and sigma values. The tree that was supposed to be produced is  
![image](https://github.com/JuliaPOMDP/POMDPs.jl/assets/78047777/22753b65-3420-4167-b47f-6244355e6841)

This was presented in the discussion for the ARDESPOT solver (https://github.com/JuliaPOMDP/ARDESPOT.jl). I was trying to reproduce it so I could see if I could replicate it in my model.

Thank you.

**Reply by zsunberg** [reply]
Yeah, I think the error is being raised when your code calls `info[:tree]`, so I am trying to figure out what type of object `info` ends up being. Is it `nothing` or is it a `Dict`, and if it is a `Dict`, then what keys does it have in it?

**Reply by Etwari3** [reply]
I ran a BasicPOMCP solver and got it to run.
![image](https://github.com/JuliaPOMDP/POMDPs.jl/assets/78047777/af67289a-0ab1-4674-abbb-ca10cb1130df)
It looks like a Dict with :tree_queries ,   :search_time_us and   :tree 

I am not sure it is the same for ARDESPOT

**Reply by zsunberg** [reply]
I think you should check what it is for DESPOT. Comment out the `inchrome` line and add `display(info)`.
